public-api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java

```diff
@@ -1,23 +1,23 @@
 package org.pmiops.workbench.cdr.dao;
 
-import org.pmiops.workbench.cdr.model.CBCriteria;
+import org.pmiops.workbench.cdr.model.DbCriteria;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.data.repository.query.Param;
 
 import java.util.List;
 
-public interface CBCriteriaDao extends CrudRepository<CBCriteria, Long> {
+public interface CBCriteriaDao extends CrudRepository<DbCriteria, Long> {
     @Query(value = "select c.id, c.parent_id, c.domain_id, c.is_standard, c.type, c.subtype, c.concept_id, c.code, c.name, c.value, c.est_count, c.is_group, c.is_selectable, c.has_attribute, c.has_hierarchy, c.has_ancestor_data, c.path, c.synonyms, co.can_select" +
             " from cb_criteria c join concept co on c.concept_id=co.concept_id where c.concept_id = :conceptId and match(c.synonyms) against(:matchWord in boolean mode) and c.domain_id=:domain\n" +
             "union all\n" +
             "select c.id, c.parent_id, c.domain_id, c.is_standard, c.type, c.subtype, c.concept_id, c.code, c.name, c.value, c.est_count, c.is_group, c.is_selectable, c.has_attribute, c.has_hierarchy, c.has_ancestor_data, c.path, c.synonyms, co.can_select\n" +
             " from cb_criteria c join concept co on c.concept_id=co.concept_id where parent_id in (select distinct id from cb_criteria where concept_id=:conceptId and match(synonyms) against(:matchWord in boolean mode) and domain_id=:domain)\n" +
             "order by cast(est_count as unsigned) desc", nativeQuery = true)
-    List<CBCriteria> findParentCounts(@Param("conceptId") String conceptId,@Param("domain") String domain,@Param("matchWord") String matchWord);
+    List<DbCriteria> findParentCounts(@Param("conceptId") String conceptId,@Param("domain") String domain,@Param("matchWord") String matchWord);
```

Comment:
If writing test cases for dao calls that use match(), then you will need to make sure you have this class: https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/cdr/CommonTestDialect.java#L9
Also, need this in application.properties in src/test/resources: https://github.com/all-of-us/workbench/blob/master/api/src/test/resources/application.properties#L6